### DISCUSSION TITLE
Github API - Revert endpoint

### DISCUSSION THREAD

**Discussion by angeliski** [community/community#36054] [Pull Requests]
I am trying to found the endpoint how enables us to create a Revert Pull Request.

I can do this using the UI, but what I want is do using the Rest API, is there any prevision to release that?

**Comment by williamwang-rbx** [comment]
Looks like this is now possible using GraphQL: https://github.blog/changelog/2023-01-27-api-for-reverting-a-pull-request/.
if this answer is correct: https://stackoverflow.com/questions/56602936/how-to-use-github-api-v3-to-revert-a-pull-request-or-revert-a-commit-not-from, then it is still not possible using REST API.

**Comment by Noumanluckey8057** [comment]
PiOS License

Copyright (C) <year> <copyright holder>

Permission is hereby granted by the application software developer (â€œSoftware Developerâ€), free
of charge, to any person obtaining a copy of this application, software and associated 
documentation files (the â€œSoftwareâ€), which was developed by the Software Developer for use on 
Pi Network, whereby the purpose of this license is to permit the development of derivative works 
based on the Software, including the right to use, copy, modify, merge, publish, distribute, 
sub-license, and/or sell copies of such derivative works and any Software components incorporated 
therein, and to permit persons to whom such derivative works are furnished to do so, in each case, 
solely to develop, use and market applications for the official Pi Network. For purposes of this 
license, Pi Network shall mean any application, software, or other present or future platform 
developed, owned or managed by Pi Community Company, and its parents, affiliates or subsidiaries, 
for which the Software was developed, or on which the Software continues to operate.  However, 
you are prohibited from using any portion of the Software or any derivative works thereof in any 
manner (a) which infringes on any Pi Network intellectual property rights, (b) to hack any of Pi 
Networkâ€™s systems or processes or (c) to develop any product or service which is competitive with 
the Pi Network.
 
The above copyright notice and this permission notice shall be included in all copies or 
substantial portions of the Software.

THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, 
INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE 
AND NON-INFRINGEMENT. IN NO EVENT SHALL THE AUTHORS, PUBLISHERS, OR COPYRIGHT HOLDERS OF THIS 
SOFTWARE BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY OR CONSEQUENTIAL 
DAMAGES (INCLUDING, BUT NOT LIMITED TO BUSINESS INTERRUPTION, LOSS OF USE, DATA OR PROFITS) 
HOWEVER CAUSED AND UNDER ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR 
TORT (INCLUDING NEGLIGENCE) ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE 
OR OTHER DEALINGS IN THE SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGES.

Pi, Pi Network and the Pi logo are trademarks of the Pi Community Company.
